lib/kuksa_val_v2/src/lib.rs

```diff
@@ -0,0 +1,1508 @@
+/********************************************************************************
+* Copyright (c) 2025 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+* information regarding copyright ownership.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License 2.0 which is available at
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* SPDX-License-Identifier: Apache-2.0
+********************************************************************************/
+pub use databroker_proto::kuksa::val as proto;
+use databroker_proto::kuksa::val::v2::signal_id::Signal::Path;
+use databroker_proto::kuksa::val::v2::{
+    BatchActuateRequest, GetServerInfoRequest, GetValuesRequest, Metadata,
+    OpenProviderStreamResponse, SignalId, Value,
+};
```

Comment:
either remove use databroker_proto_kuksa::val as proto 5 lines above. or put this into use proto:v2: in line 25